dotCMS/src/main/java/com/dotcms/publisher/util/dependencies/DependencyModDateUtil.java

```diff
@@ -0,0 +1,279 @@
+package com.dotcms.publisher.util.dependencies;
+
+import com.dotcms.publisher.assets.bean.PushedAsset;
+import com.dotcms.publisher.assets.business.PushedAssetsCache;
+import com.dotcms.publisher.bundle.bean.Bundle;
+import com.dotcms.publisher.endpoint.bean.PublishingEndPoint;
+import com.dotcms.publisher.environment.bean.Environment;
+import com.dotcms.publisher.pusher.PushPublisherConfig;
+import com.dotcms.publisher.util.PusheableAsset;
+import com.dotcms.publishing.PublisherConfig;
+import com.dotcms.publishing.PublisherConfig.Operation;
+import com.dotcms.publishing.PublisherConfiguration;
+import com.dotcms.util.AnnotationUtils;
+import com.dotmarketing.beans.VersionInfo;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.business.Versionable;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.containers.model.Container;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.contentlet.model.ContentletVersionInfo;
+import com.dotmarketing.portlets.folders.model.Folder;
+import com.dotmarketing.portlets.languagesmanager.model.Language;
+import com.dotmarketing.portlets.links.model.Link;
+import com.dotmarketing.portlets.rules.model.Rule;
+import com.dotmarketing.portlets.structure.model.Relationship;
+import com.dotmarketing.portlets.structure.model.Structure;
+import com.dotmarketing.portlets.templates.model.Template;
+import com.dotmarketing.portlets.workflows.model.WorkflowScheme;
+import com.dotmarketing.util.InodeUtils;
+import com.dotmarketing.util.Logger;
+import java.util.stream.Collectors;
+import org.apache.commons.lang.StringUtils;
+
+import java.util.*;
+
+/**
+ * Provide util method
+ */
+public class DependencyModDateUtil extends HashSet<String> {
```

Comment:
Extending from HashSet not needed anymore

Reply:
done https://github.com/dotCMS/core/pull/20521/commits/078b38ddfdf91f36ed147df16700c696a0da57c1#diff-4620da1e420b91c427e5941f3851da9b7ea5ee39096809650bd047558ef5b2d0L13